### ISSUE TITLE
 the accuracy has not changed or improved. Why?

### ISSUE THREAD

**Issue by starstars123** [ZhiGaomcislab/CurvatureGeneration_FSL#2]
Hello, the data set is miniimagenet. When the program runs to 155 epochs, the accuracy on the test set is still 0.2586. The loss is backward, but the accuracy has not changed or improved. Why?
![%GZ)$Y(HWM~2SEUYRIQM_PE](https://user-images.githubusercontent.com/66480147/197478507-1e72ab32-427a-4aa6-9044-d7ea1f38d267.png)

**Comment by zhigao2017** [comment]
> Hello, the data set is miniimagenet. When the program runs to 155 epochs, the accuracy on the test set is still 0.2586. The loss is backward, but the accuracy has not changed or improved. Why? ![%GZ)$Y(HWM~2SEUYRIQM_PE](https://user-images.githubusercontent.com/66480147/197478507-1e72ab32-427a-4aa6-9044-d7ea1f38d267.png)

Hi. Which code and which setting do you run?

**Comment by zhigao2017** [comment]
We updated the code last month. Do you use the latest code?

**Comment by starstars123** [comment]
> We updated the code last month. Do you use the latest code?

yes, I use the latest code ,  The running files and configurations are as follows：
![YEN$L{}} )7K7%O5%1FE)3D](https://user-images.githubusercontent.com/66480147/197483063-3e52588f-63d2-4558-9578-fb2693121b8b.png)

**Comment by zhigao2017** [comment]
> > We updated the code last month. Do you use the latest code?
> 
> yes, I use the latest code , The running files and configurations are as follows： ![YEN$L{}} )7K7%O5%1FE)3D](https://user-images.githubusercontent.com/66480147/197483063-3e52588f-63d2-4558-9578-fb2693121b8b.png)

Hi, you can try the following hyperparameters. 
![image](https://user-images.githubusercontent.com/38682784/197661664-69e76446-4b88-42b4-ac11-68e3f2423380.png)

**Comment by starstars123** [comment]
> > > We updated the code last month. Do you use the latest code?
> > 
> > 
> > yes, I use the latest code , The running files and configurations are as follows： ![YEN$L{}} )7K7%O5%1FE)3D](https://user-images.githubusercontent.com/66480147/197483063-3e52588f-63d2-4558-9578-fb2693121b8b.png)
> 
> Hi, you can try the following hyperparameters. ![image](https://user-images.githubusercontent.com/38682784/197661664-69e76446-4b88-42b4-ac11-68e3f2423380.png)

Thank you for your reply. After I try your parameters and run the program for 200 epochs, the accuracy has changed. Excuse me, how do I want to run this model on my own private dataset and find the appropriate parameters?
![QWZMY I6%KEHP DMINZ}4$C](https://user-images.githubusercontent.com/66480147/199135962-7595f337-d44f-41db-bfcc-9905be6c5968.png)

**Comment by zhigao2017** [comment]
Hi, for ConvNet, it needs more epochs to train the model. Besides, maybe tuning parameters about the curvature generation processes is useful. Such as 'l' and 'divide'.
